src/main/java/org/graylog/plugins/usagestatistics/collectors/ElasticsearchCollector.java

```diff
@@ -69,25 +69,17 @@ public ElasticsearchCollector(Client client, ClusterStatsService clusterStatsSer
                 continue;
             }
 
-            final MacAddress macAddress = info.getNetwork() == null ? MacAddress.EMPTY : MacAddress.create(info.getNetwork().primaryInterface().macAddress());
-            final HostInfo.Cpu cpu = info.getOs().cpu() == null ? null : HostInfo.Cpu.create(
-                    info.getOs().cpu().model(),
-                    info.getOs().cpu().vendor(),
-                    info.getOs().cpu().mhz(),
-                    info.getOs().cpu().totalCores(),
-                    info.getOs().cpu().totalSockets(),
-                    info.getOs().cpu().coresPerSocket(),
-                    info.getOs().cpu().cacheSize().bytes()
-            );
-            final HostInfo.Memory memory = info.getOs().mem() == null ? null : HostInfo.Memory.create(info.getOs().mem().total().bytes());
-            final HostInfo.Memory swap = info.getOs().swap() == null ? null : HostInfo.Memory.create(info.getOs().swap().total().bytes());
+            final MacAddress macAddress = MacAddress.EMPTY;
+            final HostInfo.Cpu cpu = null;
+            final HostInfo.Memory memory = null;
+            final HostInfo.Memory swap =  null;
             final HostInfo hostInfo = HostInfo.create(macAddress, cpu, memory, swap);
```

Comment:
:white_check_mark: Fair enough. Maybe add a `@Deprecated` or at least a `TODO` comment to remove this in a future version.